### PR TITLE
blog updates

### PR DESCRIPTION
**Pull request by teddymaulana**

### CODE REVIEW

File: docs/templates/article.html [hunk 1]
```diff
@@ -13,37 +13,67 @@
...
-						<li class="list-inline-item">
-							<a href="#"
-								class="text-decoration-none sni sni__facebook-square text-primary social-icon-primary"></a>
-						</li>
-						<li class="list-inline-item">
-							<a href="#"
-								class="text-decoration-none sni sni__pinterest-square text-primary social-icon-primary"></a>
-						</li>
-					</ul>
 					<div class="article__content">
```
Comment (foria) [inline docs/templates/article.html:77]: we don't need to wrap the content inside this DIV, please remove

File: docs/templates/article.html [hunk 2]
```diff
@@ -66,6 +96,19 @@ <h3 class="mb-1">5. Give it time to do its thing</h3>
...
+						<li class="list-inline-item">
+							<a href="#" class="text-decoration-none sni sni__facebook-square text-primary social-icon-primary"></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="text-decoration-none sni sni__pinterest-square text-primary social-icon-primary"></a>
+						</li>
+					</ul>
 				</div>
 
 				<div class="blog-post-grid__sidebar col-12 d-lg-flex mt-2 mt-lg-0 mb-3 mb-lg-0 sticky-bottom__container">
```
Comment (foria) [inline docs/templates/article.html:114]: use tag SECTION for comments section

File: docs/templates/article.html [hunk 3]
```diff
@@ -66,6 +96,19 @@ <h3 class="mb-1">5. Give it time to do its thing</h3>
...
+							<a href="#" class="text-decoration-none sni sni__twitter-square text-primary social-icon-primary"></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="text-decoration-none sni sni__facebook-square text-primary social-icon-primary"></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="text-decoration-none sni sni__pinterest-square text-primary social-icon-primary"></a>
+						</li>
+					</ul>
 				</div>
```
Comment (foria) [inline docs/templates/article.html:113]: please review the integration of the section SIDEBAR as it's done in compounds, we must use the same code adding additional classes for the position. sticky-bottom__container is not used as also .d-lg-flex. Please review all the classes used and remove the class and code not used.

**Review by foria — CHANGES_REQUESTED** [review]
please review code. careful about syntax and indexation.
there is some optimisation to do on the code, we might brought something wrong from the previous version.
Please review my comments, apply the adjustments to all components/templates involved and update the Shopify theme as well

File: docs/templates/article.html (hunk 3 above)
Comment (teddymaulana) [reply docs/templates/article.html:109]: I am not understanding this one, also I am not really sure what has been done on the compound?
the sticky-bottom__container is used to make it sticky

**Pull request closed without merging**